pyramid/tier_fs.go

```diff
@@ -176,6 +176,11 @@ func (tfs *TierFS) store(namespace, originalPath, filename string) error {
 	}
 }
 
+// removeTempFile removes the closed non-stored file from the FS.
+func (tfs *TierFS) removeTempFile(originalPath string) error {
+	return os.Remove(originalPath)
+}
```

Comment:
why do we need this method? we can just call the os.Remove

Reply:
Done.